bears/jinja2/Jinja2Bear.py

```diff
@@ -0,0 +1,335 @@
+                    end_line=line_number,
+                    end_column=m.end("label") + 1,
+                    diffs=diff)
+
+    def check_for_empty_control_stack(self, filename):
+        '''
+        Checks if the control stack is empty and yields a Result for each
+        item left on the stack.
+
+        :param filename: The name of the file currently being inspected
+        '''
+        if len(self.control_stack) > 0:
+            for (_, line_number) in self.control_stack:
+                yield Result.from_values(
+                    origin=self,
+                    message="Control start tag has no corresponding end",
+                    file=filename,
+                    line=line_number,
+                    severity=RESULT_SEVERITY.MAJOR)
+
+    def run(self,
+            filename,
+            file,
+            variable_spacing: int=1,
+            control_spacing: int=1):
+        '''
+        Check Jinja2 templates for syntax, formatting and documentation issues.
+        The following aspects are being looked at:
+
+        * Variable spacing:    Variable tags should be padded with one space on
+                               each side, like this: ``{{ var_name }}``. This
+                               can be set to any number of spaces via the
+                               setting variable_spacing.
+                               Malformatted variable tags are detected and
+                               fixes suggested.
+        * Control spacing:     Like variable spacing, but for control blocks,
+                               i.e. if and for constructs. Looks at both start
+                               and end block.
+        * Control labels:      It is good practise to label the end of an if or
+                               for construct with a comment equal to the
+                               content of the start, like so::
+
+                                    {% for x in y %}
+                                      do something
+                                    {% endfor %}{# for x in y %}
+
+                               Mising or differing labels are detected and
+                               fixes suggested.
+                               Constructs with start and end on the same line
+                               are being ignored.
+        * unbalanced blocks:   Each opening tag for a for or if construct must
+                               be closed by a corresponding end tag. An
+                               unbalanced number of opening and closing tags is
+                               invalid syntax and will be reported with MAJOR
+                               severity by the bear.
+
+
+        :param variable_spacing: The number of spaces a variable block
+                                 should be spaced with. Default is 1.
+        :param control_spacing:  The number of spaces a control block
+                                 should be spaced with. Default is 1.
```

Comment:
bump, please use the one lasse mentioned :D 

check example here to see better http://docs.coala.io/en/latest/Developers/Codestyle.html